src/scenes/pt.cpp

```diff
@@ -50,15 +53,19 @@ void PathTracer::render(Scene const &scene, Camera const &cam, AppContext &ctx,
     if (not rSample.isZero()) {
       Radiance change = rSample.cwiseQuotient(radianceBuffer[idx]);
       if (not change.hasNaN()) {
-        avgChange += change;
+        avgChangeR += change[0];
```

Comment:
As mentioned, just moving maxCoeff here should be fine. The calculation is not exact anyway at this point.